OneSignalExample/Assets/OneSignal/Example/OneSignalExampleScene.unity

```diff
@@ -2496,7 +2708,7 @@ MonoBehaviour:
   email: 
   externalId: 
   phoneNumber: 
-  appId: 
+  appId: 77e32082-ea27-42e3-a898-c72e141824ef
```

Comment:
I think you should remove the app Id here

Reply:
nice catch, removed!